app/policies/rent_policy.rb

```diff
@@ -0,0 +1,30 @@
+class RentPolicy < ApplicationPolicy
+  class Scope
+    attr_reader :user, :scope
+
+    def initialize(user, scope)
```

Comment:
The ApplicacionPolicy already declares this initializer, if you are not going to change the behaviour don't write it again